cpm

```diff
@@ -257,6 +257,20 @@ _xbps() {
     esac
 }
 
+_slackpkg() {
+    case "$OP" in
+        install) $SUDO slackpkg install "$@";;
+        remove) $SUDO slackpkg remove "$@";;
+        list) ls -l /var/log/packages/;;
+        count) ls -l /var/log/packages/ | tot;;
+        update) $SUDO slackpkg update gpg && slackpkg update;;
+        upgrade) $SUDO slackpkg upgrade-all;;
+        search) slackpkg search $1;;
+        show) slackpkg Info $1;;
```

Comment:
Should this be `Info` or `info`?

Reply:
the wiki [here]https://docs.slackware.com/slackware:slackpkg#info(url) says `slackpkg Info packagename`. thinking back, i think thats a typo, because [here]https://docs.slackware.com/howtos:slackware_admin:querying_installed_packages(url) says `slackpkg info packagename`. ill change that in just a second

Reply:
4484fb5 changes `Info` to be `info`

Reply:
(lol trusting the wiki > normal common sense and logic)